README.md

```diff
@@ -0,0 +1,107 @@
+# API Advisor
+
+```json
+BaseURL: https://rain-forecast-risk-alert.herokuapp.com/
+```
+
+## Users
+
+### **POST** /api/users/signup
+
+Register a new user.
+
+- Not authenticated
+
+## Parameter
+
+| **Field** | **Type** | **Description**          |
+| --------- | -------- | ------------------------ |
+| name      | string   | User Name                |
+| email     | string   | User email               |
+| phone     | string   | User phone number        |
+| cep       | string   | User location cep number |
+| city      | string   | User city location       |
+
+**Body:**
+
+```json
+{
+  "name": "Danilo Motta",
+  "email": "d.motta@gmail.com",
+  "phone": "24976520981",
+  "cep": "25651-079",
+  "city": "Petrópolis"
+}
+```
+
+Success-Response:
+
+**HTTP/1.1**: 201 **CREATED**
+
+```json
+{
+  "name": "Danilo Motta",
+  "email": "d.motta@gmail.com",
+  "phone": "24976520981",
+  "address": "25651-079",
+  "city": "Petrópolis",
+  "state": "Rio de Janeiro"
+}
+```
+
+Error 4xx
```

Comment:
In one of the readmes i made, i've found a way to create a toggle to insert those errors, i think it's more organized that way, if everyone agrees we can apply that here too!

Reply:
Done.